ompi/mca/pml/ob1/pml_ob1_sendreq.c

```diff
@@ -125,8 +125,16 @@ static int mca_pml_ob1_send_request_free(struct ompi_request_t** request)
     return OMPI_SUCCESS;
 }
 
+#if MPI_VERSION >= 4
+extern int mca_pml_cancel_send_callback(struct ompi_request_t* request, int complete);
```

Comment:
No need to define it here again, already defined in a header: https://github.com/open-mpi/ompi/pull/10625/files#diff-0ec27ebce1ba518c96e6215373b7f65cc9e7db9a153c2df144cc6bc17fd293a2R388